### ISSUE TITLE
Prints Warning

### ISSUE THREAD

**Issue by SeanPedersen** [wookayin/gpustat#94]
Love this package but I am getting an annoying warning. Hope it can be fixed.

```
$ python -m gpustat                                          
/usr/lib/python3.9/runpy.py:127: RuntimeWarning: 'gpustat.__main__' found in sys.modules after import of package 'gpustat', but prior to execution of 'gpustat.__main__'; this may result in unpredictable behaviour
  warn(RuntimeWarning(msg))
archlinux            Sat Jan  9 14:13:46 2021  460.32.03
[0] GeForce MX150    | 35'C,   0 % |  1191 /  2002 MB | sean(1189M)
```

**Comment by wookayin** [comment]
This should not happen if you run `$ gpustat`, but yes it does happens with `python -m gpustat`. Let me take a look at how we can fix this.

**Comment by SeanPedersen** [comment]
Actually I am not using python -m hypertag anymore so for it is no issue anymore. Just thought I should let u know.

**Comment by wookayin** [comment]
This already has been fixed in 5c05c3c. I just need to deliver a GA release.
